README.md

```
@@ -140,59 +154,89 @@ node agent/scripts/validate-ideas-urls.js
 ```bash
 git clone https://github.com/your-username/gsoc-2026-org-finder.git
 cd gsoc-2026-org-finder
 ```
 
-### 2. Add GitHub Token (for live stats + Good First Issues)
-In your Vercel dashboard → Project Settings → Environment Variables:
+### 2. Add GitHub Token
+
+Required for live stats and Good First Issues. In your Vercel dashboard go to **Project Settings → Environment Variables** and add:
+
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)